terraform/modules/hub/ingress.tf

```diff
@@ -177,6 +177,27 @@ resource "aws_lb_listener" "ingress_https" {
   }
 }
 
+resource "aws_lb_listener_rule" "ingress_root" {
+  listener_arn = "${aws_lb_listener.ingress_https.arn}"
+  priority     = 140
```

Comment:
thank you for not picking 130 and colliding with #161 😁

Reply:
seemed only right given I stole the rest of it off you